components/Article/Page.js

```diff
@@ -368,6 +368,7 @@ class ArticlePage extends Component {
         formatColor={formatColor}
         audioSource={audioSource}
         audioCloseHandler={this.toggleAudio}
+        standalone={runningInApp}
```

Comment:
Couldn't find a better name for this prop. Seemed incorrect to me to make the `Frame` receive a prop called `runningInApp`, but I can change it easily if you want. `frameless` may be a valid option, or we can have two different props, one for `header` and other for `footer`